R/plotly_methods.R

```diff
@@ -92,7 +92,6 @@
 #' @return A plotly
 #' @export
 #' @examples
-#' \dontrun{
```

Comment:
in my experience plotly does not play well with remote tests, I think it is fair enough to leave it out as norun.

Most servers do not have javascript libraries needed for interactive plotting

Reply:
the norun, is taken from plotly library itself

Reply:
Here you go :) the test fail. I will revert this edit
```
  > # plot_ly() tries to create a sensible plot based on the information you 
  > # give it. If you don't provide a trace type, plot_ly() will infer one.
  > plot_ly(economics, x = ~pop)
  Error in plot_ly(economics, x = ~pop) : object 'economics' not found
  Execution halted
```

Reply:
Ok fine, I think it's still failing because the Rd needs reverting too. Are you able to do that? Or want me to?

Reply:
> 
> 
> Ok fine, I think it's still failing because the Rd needs reverting too. Are you able to do that? Or want me to?

I'll do it no worries